src/analysis/convex/krein_milman.lean

```diff
@@ -56,7 +56,8 @@ See chapter 8 of [Barry Simon, *Convexity*][simon2011]
 open set
 open_locale classical
 
-variables {E : Type*} [normed_add_comm_group E] [normed_space ℝ E] {s : set E}
+variables {E : Type*} [add_comm_group E] [module ℝ E] [topological_space E] [t2_space E]
+  [topological_add_group E] [has_continuous_smul ℝ E] [locally_convex_space ℝ E] {s : set E}
```

Comment:
Please change the docstring below for Krein-Milman and remove the Todo in the beginning of the file.